code/api/src/modules/user/tests/mutations/userMutations.test.js

```diff
@@ -0,0 +1,25 @@
+import request from 'supertest';
+import testServer from '../../../../utils/test_helper/test_server'
+
+describe('User', () => {
+  let server;
+  beforeAll(() => {
+    server = testServer;
+  })
+  it('can edit a users email', async (done) => {
+    const response = await request(server)
+    .post('/graphql')
+    .send({query: `mutation {
```

Comment:
This syntax is what is needed to test a mutation.

Reply:
I'd love to go over this in person.